src/app/components/query-explainer/query-explainer.types.ts

```diff
@@ -0,0 +1,41 @@
+export interface QueryExplainerResponse {
+    input: {
+        query: string;
+        queryDialect: string;
+        dataFormat: string;
+        schemaFormat: string;
+        include: string[];
+        skip: number;
+        limit: number;
+        datasets: [{ alias: string; blockHash: string; id: string }];
+    };
+    commitment: {
+        inputHash: string;
+        outputHash: string;
+        subQueriesHash: string;
+    };
+    proof: {
+        proofValue: string;
+        type: string;
+        verificationMethod: string;
+    };
+    output?: {
+        data: [string[]];
+        dataFormat: string;
```

Comment:
And this definitely is an enum

Reply:
Changed, set ` dataFormat: keyof typeof DataBatchFormat;`